assets/js/vue/FileManager.vue

```diff
@@ -108,7 +101,17 @@ const deleteItem = (item) => {
                     reject(error)
                 })
         } else {
-            reject(new Error('Cannot delete folders'));
+            axiosInstance
+                .get(`${Routing.generate('pelagos_api_get_file_ids_dataset_submission')}/${datasetSubmissionId}?path=${item.path}`)
+                .then(response => {
+                    response.data.forEach(id => {
+                        axiosInstance.delete(`${Routing.generate('pelagos_api_file_delete')}/${id}`);
```

Comment:
Why not have an API that deletes files,  rather than this handoff, it will just slow things down.

Reply:
You can even call a route from the controller itself!

Reply:
So, you are saying have a delete folder API, which takes in multiple file ids? We definitely would need the getFileIDs API, as we would do the same thing for moving/renaming folders as well